test/e2e/specs/test.spec.js

```diff
@@ -45,15 +95,37 @@ describe('Vaults UI Test Cases', () => {
         cy.contains(
           'p',
           'You can manage your vaults from the "My Vaults" view.',
+          { timeout: MINUTE_MS },
         ).should('exist');
+        cy.contains('Manage my Vaults').click();
       });
     });
 
-    it('should adjust the collateral by performing a withdrawl and approve the transaction successfully', () => {
-      cy.contains('button', 'Manage my Vaults').click();
-      cy.contains('button', 'Back to vaults').click();
-      cy.contains('div', /ATOM.*#8/).click();
+    it('should open the new vault', () => {
+      cy.get('span')
+        .contains(/My Vaults.*\(\d+\)/)
+        .children()
+        .first()
+        .spread(element => {
+          // Get the total number of vaults present
```

Comment:
Why use `spread`? `spread` is useful when we have an array of elements to process? And over here we only process the `.first()`? Do you think we should use `.then()`?